src/main/java/application/controllers/MainController.java

```diff
@@ -132,9 +179,16 @@ public void createMenu() {
 
     /**
      * Switches the scene to the graph view.
+     *
+     * @param delta the diff in view to apply.
      */
-    public void switchScene() {
+    public void switchScene(int delta) {
+        currentView += delta;
+        currentView = Math.max(0, currentView);
+
+        currentView = Math.min(9, currentView);
```

Comment:
Please add a comment that explains what happens with the altering of 'currentView'.